### PR TITLE
Improve some inappropriate descriptions in the docstring of DinoTransformerDecoder.

### PR DESCRIPTION
**Pull request by keyakiluo**
Improve some inappropriate descriptions in the docstring of DinoTransformerDecoder.

### CODE REVIEW

**Comment by keyakiluo** [comment]
@Li-Qingyun @Czm369

File: mmdet/models/layers/transformer/dino_layers.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def forward(self, query: Tensor, value: Tensor, key_padding_mask: Tensor,
                 spatial_shapes: Tensor, level_start_index: Tensor,
                 valid_ratios: Tensor, reg_branches: nn.ModuleList,
                 **kwargs) -> Tensor:
```
Comment (Li-Qingyun) [inline mmdet/models/layers/transformer/dino_layers.py:30]: It seems that the typehint is incorrect.

**Review by Li-Qingyun — APPROVED** [review]
LGTM after modifying typehint.

**Comment by hhaAndroid** [comment]
@keyakiluo Please fix the cl

**Comment by keyakiluo** [comment]
Some formatting issues have been fixed. A review is required for subsequent checks. @hhaAndroid @Czm369 @Li-Qingyun

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**